notebooks/06_validate.ipynb

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    for id in items_missing_annotation:
</pre><p><code>id</code> is also the Python's pre-defined keyword.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/785/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>